src/DataSource/Types.hs

```diff
@@ -294,12 +298,27 @@ newtype CommentOuter = CommentOuter {
 
 -- | Zendesk ticket
 data Ticket = Ticket
-    { tComment :: !Comment
+    { tComment     :: !Comment
     -- ^ Ticket comment
-    , tTag     :: ![Text]
+    , tTag         :: !TicketTags
+    , tField       :: ![TicketField]
```

Comment:
I'm not entirely convinced that we need this. Let's have a chat first.

Reply:
Sure!